README.md

```diff
@@ -1 +1,10 @@
-# microverse-test
\ No newline at end of file
+Hello Microverse Project.
+
+
+####
+I created this project to learn how to use linters.
+
+
+LANGUAGES USED.
+1. HTML
+2. CSS
```

Comment:
Please, check the next link: [README Template provided by Microverser](https://github.com/microverseinc/readme-template) and feel free to use this template to make a more professional README file, also, you can check the next link: [Project Sample](https://github.com/DanteAlonsoHT/Leaderboard_API) to check what we expect from students 😉 

- [ ] Instead of your current README file, use the Template provided by Microverse to make an impressive README file 😉 Remember that you can remove some unnecessary sections from the Template provided by Microverse such as "Run tests" and "Deployment".